exercises/C-comparison-operators/exercise.js

```diff
@@ -8,13 +8,16 @@
 var studentCount = 16;
 var mentorCount = 9;
 var moreStudentsThanMentors; // finish this statement
+moreStudentsThanMentors = studentCount > mentorCount;
 
 var roomMaxCapacity = 25;
 var enoughSpaceInRoom; // finish this statement
+enoughSpaceInRoom = roomMaxCapacity >= studentCount + mentorCount;
```

Comment:
Great Job so far Somayeh. Just to make the code shorter as much as possible - you can use 
enoughSpaceInRoom = roomMaxCapacity>=25.. rather than repeating since it's more or less saying the same thing